src/server/auth/prismaAdapter.ts

```diff
@@ -11,7 +11,7 @@ type CreateUser = (user: PrismaUser) => Awaitable<TrimmedUser>;
 
 const getCreateUser = (prisma: PrismaClient): CreateUser => {
     return (user: PrismaUser): Awaitable<TrimmedUser> => {
-        const { id, ...rest } = user;   //remove the discord id from the user object - a replacement is auto-generated by Prisma
+        const { id, email, emailVerified, ...rest } = user;   //remove the discord id from the user object - a replacement is auto-generated by Prisma
```

Comment:
Not type safe, but we need this to make auth work?

Reply:
Yeah it looks that way. It's probably fixable with a custom provider, but I don't think it's a very big deal.